src/components/Combobox.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="combobox">
+    <input
+      type="text"
+      :id="id"
+      class="combobox__input"
+      ref="comboboxInput"
+      :value="value"
+      :placeholder="placeholder"
+      @keydown="checkInput"
+      @input="$emit('input', $event.target.value)"
+      autocomplete="off"
+    />
+    <ul v-if="source" class="combobox__options" ref="comboboxList">
+      <li
+        v-for="(option, index) in source"
+        :key="index"
+        class="combobox__option"
+      >
+        {{ option }}
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import Combobo from 'combobo';
+
+export default {
+  name: 'Combobox',
+  props: {
+    id: String,
+    source: Array,
+    value: String,
+    allowCustomInput: {
+      type: Boolean,
+      default: true,
+    },
+    placeholder: String,
+  },
+  methods: {
+    init() {
```

Comment:
I'd personally just move that to `mounted()` to reduce indirection.

Reply:
Haha, I was probably prematurely optimising, not sure if we'll meet a case where we'd want to init on a different lifecycle hook. Have moved to `mounted()` now.